src/FormField/PasswordInputField.tsx

```diff
@@ -22,11 +21,10 @@ type FieldProps = {
   className?: string;
   /** an optional style object to pass to top level element of the component */
   style?: React.CSSProperties;
+  /** the properties of password input */
+  passwordInputProps: PasswordInput.Props;
 };
 
-type NonDefaultProps = FieldProps &
-  ObjectOmit<PasswordInput.Props, keyof FieldProps>;
-
```

Comment:
could you double-check if removing this and using all the properties of `PasswordInput.Props` is legit? I found no conflict but I'm not 100% confident that I'm not missing anything

Reply:
looks legit 👍